idp_node/myproxy.py

```diff
@@ -231,13 +239,17 @@ def restart_myproxy_server():
 
 def myproxy_status():
     '''Checks the status of the myproxy server'''
-    if os.access("/etc/init.d/myproxy-server", os.X_OK):
-        status = esg_functions.call_subprocess("/etc/init.d/myproxy-server status")
-    print "myproxy server status:", status["stdout"]
-    if "running" in status["stdout"]:
-        return (True, status)
-    else:
+    try:
+        status = esg_functions.call_binary("/etc/init.d/myproxy-server", ["status"])
+    except ProcessExecutionError:
+        pass
+    try:
+        status = esg_functions.call_binary("/etc/init.d/myproxy", ["status"])
+    except ProcessExecutionError:
```

Comment:
Is there a reason these aren't of the form `service myproxy status`?

Reply:
Okay, I did some reading on stackoverflow and it looks like we should use service to be more future-proof.

Reply:
Just pushed out a change for it